### PR TITLE
Cypress/E2E: Fix MM-T1467 for more modal dialog accessibility

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix MM-T1467 for more modal dialog accessibility

#### Ticket Link
none, failing on daily Cypress run

Screen Shot 2020-10-01 at 12 58 14 AM

### CODE REVIEW

File: e2e/cypress/integration/accessibility/accessibility_modals_dialogs_spec.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ describe('Verify Accessibility Support in Modals & Dialogs', () => {
 
     it('MM-T1454 Accessibility Support in Different Modals and Dialog screen', () => {
         // * Verify the aria-label in main menu button
-        cy.get('#headerInfo button').should('have.attr', 'aria-label', 'main menu');
+        cy.get('#headerInfo button', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible').and('have.attr', 'aria-label', 'main menu');
```
Comment (saturninoabril) [inline e2e/cypress/integration/accessibility/accessibility_modals_dialogs_spec.js:66]: Not related but got intermittent failing here while running in local.

File: e2e/cypress/integration/accessibility/accessibility_modals_dialogs_spec.js [hunk 2]
```diff
@@ -126,51 +126,60 @@ describe('Verify Accessibility Support in Modals & Dialogs', () => {
...
+
+                        // * Verify the accessibility support in search input
+                        cy.get('#searchChannelsTextbox').should('have.attr', 'placeholder', 'Search channels');
+
+                        cy.waitUntil(() => cy.get('#moreChannelsList').then((el) => {
+                            return el[0].children.length === 2;
+                        }));
+
+                        // # Focus on the Create Channel button and TAB twice
+                        cy.get('#createNewChannel').focus().tab().tab();
```
Comment (saturninoabril) [inline e2e/cypress/integration/accessibility/accessibility_modals_dialogs_spec.js:164]: Main fix is here. Before doing tabs, ensure that the list is already loaded with channels.

File: e2e/cypress/integration/accessibility/accessibility_modals_dialogs_spec.js [hunk 3]
```diff
@@ -126,51 +126,60 @@ describe('Verify Accessibility Support in Modals & Dialogs', () => {
...
                         });
 
-                        // * Press Tab and verify if focus changes to Join button
+                        // * Press Tab again and verify if focus changes to next row
                         cy.focused().tab();
-                        cy.get('.more-modal__actions button').should('have.class', 'a11y--active a11y--focused');
-
-                        // * Verify previous button should no longer be focused
-                        cy.get('.more-modal__details button').should('not.have.class', 'a11y--active a11y--focused');
+                        cy.findByLabelText(getChannelAriaLabel(otherChannel)).should('have.class', 'a11y--active a11y--focused');
```
Comment (saturninoabril) [inline e2e/cypress/integration/accessibility/accessibility_modals_dialogs_spec.js:181]: Simplify check by using `cy.findByLabelText`.

**Review by srkgupta — APPROVED** [review]
Tested and passed. Changes LGTM. Thanks @saturninoabril

**Review by prapti — APPROVED** [review]
LGTM! Tests passed locally.

**Pull request merged**